package.json

```diff
@@ -4,29 +4,29 @@
   "description": "%CUSTOM_PLUGIN_SERVICE_DESCRIPTION%",
   "main": "dist/index.js",
   "scripts": {
-    "checkonly": "! grep -R '\\.only' tests/",
-    "coverage": "npm run unit -- --coverage-report=text-summary",
-    "postcoverage": "tap --coverage-report=lcov --no-browser",
+    "checkonly": "! grep -R '\\\\.only' __tests__/",
```

Comment:
Can we verify that we actually need four backslashes? Do they properly work on unix systems such as macos and linux?

Reply:
Actually checkonly is not needed with jest since `.only` is natively supported

```suggestion
```

Reply:
Done, removed script